tests/test_DGEList.py

```diff
@@ -203,11 +203,24 @@ def test_cpm():
     assert dge_list.counts[0][0] == first_pos * 1e6 / col_sum[0]
 
 
+def test_tpm():
+    dge_list = DGEList(filename=str(get_dataset_path(TEST_DATASET_NPZ)))
+    first_pos = dge_list.counts[0][0]
+    gene_lengths = np.random.randint(10, 1000, size=dge_list.genes.shape)
+    desired = (
+        1e6
+        * (first_pos / gene_lengths[0])
+        / np.sum(dge_list.counts / gene_lengths[:, None], axis=0)[0]
+    )
+    assert isinstance(first_pos, np.integer)
```

Comment:
If `numpy` is imported I think `int` is implicitly cast to `np.integer`? So no need for the NumPy type. Can just use `int`.

Reply:
Also adopted from cpm. I'll change in both